StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFlowController.swift

```diff
@@ -503,6 +512,22 @@ extension PaymentSheet {
             }
         }
 
+        func updateForWalletButtonsView() {
+            // Recreate the view controller
+            self.viewController = Self.makeViewController(
+                configuration: self.configuration,
+                loadResult: self.viewController.loadResult,
+                analyticsHelper: analyticsHelper,
+                walletButtonsShownExternally: self.walletButtonsShownExternally,
+                previousPaymentOption: self._paymentOption
+            )
+            self.viewController.flowControllerDelegate = self
+
+            // Synchronously pre-load image into cache
+            // Accessing paymentOption has the side-effect of ensuring its `image` property is loaded (e.g. from the internet instead of disk) before we call the completion handler.
+            _ = self.paymentOption
```

Comment:
There is no completion handler -- so we may load the image, but we may not report it back to anyone needing to display it?